ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1,3 +1,5 @@
+//nolint:SA1019
```

Comment:
Skip the deprecation usage lints.
<img width="1279" alt="image" src="https://github.com/user-attachments/assets/104dc84b-7960-4548-b95c-b5533b7f98c2">